org.codeberg.dnkl.foot.yml

```
@@ -0,0 +1,45 @@
+app-id: org.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: /app/bin/foot.sh
+
+finish-args:
+  - --socket=wayland
+  - --filesystem=xdg-config/foot:ro
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - name: libutf8proc
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

unless it doesn't work.